monitoring/explores/bigquery_usage_costs.explore.lkml

```diff
@@ -0,0 +1,5 @@
+include: "../views/bigquery_usage_costs.view.lkml"
+
+explore: bigquery_usage_costs {
+  from: bigquery_usage_costs
```

Comment:
```suggestion
```
Since the view and explore names are the same I believe this line is not needed. But of course, if you want to explicitly have it here that's perfectly fine.